src/CSF.SRDashboard.Client/Pages/EditNewRequest.razor.cs

```diff
@@ -97,24 +98,27 @@ public partial class EditNewRequest
 
         public bool Errorexist { get; set; }
 
-        private List<SelectListItem> languageSelections = new List<SelectListItem>();
+        private List<SelectListItem> LanguageSelections = new List<SelectListItem>();
```

Comment:
Private properties should be lower cased.
```suggestion
        private List<SelectListItem> languageSelections = new List<SelectListItem>();
```